### ISSUE TITLE
Feature request: Drag email into a bundle within the inbox

### ISSUE THREAD

**Issue by jayarr100** [leggett/simplify#643]
Not sure if this is possible (or how complicated it'd be to make happen, if so!), but it would be incredibly useful to be able to drag an email from the inbox view into a bundle within the inbox.

I've found myself instinctively trying to do that multiple times when I want to add a standalone email into an existing bundle/label. As of now, nothing happens with that action; you see the "Move 1 conversion" graphic once you start clicking and dragging, but you can't, of course, actually drop the item into a bundle.

**Comment by leggett** [comment]
I like this. I will add it to my "Someday maybe" list. But I think it is very unlikely I'll support this for a few reasons:

(1) I expect it will be very difficult to begin with. Maybe not impossible though.

(2) For bundles, ideally a label is applied with a filter. Manually labeling a thread to add it to a bundle can lead to unexpected issues in the bundle search (in:inbox label:XYZ) due to how labels work in Gmail.

Let me explain:

While you see threads in Gmail, there are no threads in the backend. There are emails with IDs that pull them together into a thread. So, labels are applied to emails, not threads.

Simplify only sees what is on the page. In the inbox, that means I see threads, not emails. I see there is a thread with two emails and Gmail shows the label XYZ on that thread. But that label may not be on all the emails in that thread. Here is why:

You've got an email that you manually applied a label to and then archived (removed the inbox label). Then a new reply came in (automatically gets the inbox label but not the XYZ label). The first email in that thread has your label but not the inbox label. The second email has the inbox label but not your label. So searching for label:XYZ in:inbox doesn't show the thread because there isn't an email in that thread with both of those labels. Remove in:inbox and you see it. 

I wrote up a massive proposal when I worked on Gmail to fix all this and I couldn't convince the backend team the issue was worth the trouble. 

Bottom line: bundles only really work well with labels applied via filter as it re-applies the label to new messages as they come in. OR it works for those that don't keep a clean inbox (remember, archiving meant the first message no longer had both the inbox and your label).

**Comment by jayarr100** [comment]
Interesting points. That totally makes sense. And thanks for the context!

I think in my mind, I was attempting to perform that action as a way to correct an instance where a certain email didn't fall within the parameters of my broad label-applying filter. But I absolutely get what you're saying about why that wouldn't be an optimal path to pursue within the current Gmail context.
